src/app/shared/services/openWeatherMaps.js

```diff
@@ -19,7 +19,7 @@ export default class OpenWeatherMaps {
         };
 
         try {
-            return this.fiveDayForecast = await this.$http.get(`${this.baseURL}/weather`, params);
+            return this.currentForecast = await this.$http.get(`${this.baseURL}/weather`, params);
```

Comment:
it's better split this line into two -> assign statement and the return statement
```
this.currentForecast = await this.$http.get(`${this.baseURL}/weather`, params);
return this.currentForecast;

Reply:
May I ask why? Already changed anyway

Reply:
Just better readability for other developers :)